apps/services/models.py

```diff
@@ -124,17 +147,28 @@ class UserRequestReview(Base):
     total_sum = Column(DECIMAL(7, 2))
     payment_deadline_date = Column(DATE)
     remark = Column(VARCHAR(length=255))
-    bed_place_id = Column(INTEGER, ForeignKey("bed_place.bed_place_id"))
+    bed_place_id = Column(
+        INTEGER,
+        ForeignKey("bed_place.bed_place_id", ondelete="CASCADE", onupdate="CASCADE"),
+    )
     reviewer = Column(
-        INTEGER, ForeignKey("user.user_id"), nullable=False
+        INTEGER,
+        ForeignKey("user.user_id", ondelete="CASCADE", onupdate="CASCADE"),
+        nullable=False,
     )  # TODO: rename column to reviewer_id
-    hostel_id = Column(INTEGER, ForeignKey("hostel.hostel_id"))
+    hostel_id = Column(
+        INTEGER,
+        ForeignKey("hostel.hostel_id", ondelete="CASCADE", onupdate="CASCADE"),
```

Comment:
ondelete value isn't correct, i think it should be do nothing or set null...

Reply:
As in the previous question. ?